### ISSUE TITLE
 Vali Loss: nan Test Loss: nan

### ISSUE THREAD

**Issue by ezelanza** [zhouhaoyi/Informer2020#511]
When I run exp.train(setting), with my own data. I've got 

Epoch: 1, Steps: 35 | Train Loss: 1.0262758 Vali Loss: nan Test Loss: nan

I'm trying to debug it but I can't see what is the problem?, the data is the same, so why am I getting a train_loss value while valid_loss = nan, when is the "same data"

I've tried with multiple normalizations, sometimes I can get values but the predictions are not good at the end. Would you help me to understand how the data should be prepared or where is the problem? I'm attaching my dataset, it has 60 features (it's already normalized)

Any comments ?
[Train_norm_full.csv](https://github.com/zhouhaoyi/Informer2020/files/11410193/Train_norm_full.csv)

**Comment by Med-Rokaimi** [comment]
I have the same problem. 
I'm using a dataset with 5 columns including the data column. the parameters are set like this:
``
args.data = 'Oil' # data
args.root_path = './data/' # root path of data file
args.data_path = 'oil.csv' # data file
args.features = 'M' # forecasting task, options:[M, S, MS]; M:multivariate predict multivariate, S:univariate predict univariate, MS:multivariate predict univariate
args.target = 'OT' # target feature in S or MS task
args.freq = 'd' # freq for time features encoding, options:[s:secondly, t:minutely, h:hourly, d:daily, b:business days, w:weekly, m:monthly], you can also use more detailed freq like 15min or 3h
args.checkpoints = './informer_checkpoints' # location of model checkpoints

args.seq_len = 96 # input sequence length of Informer encoder
args.label_len = 48 # start token length of Informer decoder
args.pred_len =1 # prediction sequence length
# Informer decoder input: concat[start token series(label_len), zero padding series(pred_len)]

args.enc_in = 4 # encoder input size
args.dec_in = 4 # decoder input size
args.c_out = 1 # output size
``

**Comment by MountVoom** [comment]
I took a look at your data and the value is very small and it's almost zero. This may cause one or more of the columns to have a variance of 0. Thus nan appears on normalization. You can check if nan exists in the data you are using, and if the variance is 0 when normalized.

**Comment by AnTi-anti** [comment]
数据中包含缺失值nan
